02-KernelModules_CharacterDevice/src/Makefile

```
@@ -0,0 +1,11 @@
+#
+# Char dev Linux kernel module
+#
+
+obj-m := intercom.o
+
+all:
+	$(MAKE) -C $(BUILD_KERNEL) M=$(CURDIR) modules
+
+clean:
+	$(MAKE) -C $(BUILD_KERNEL) M=$(CURDIR) clean
```

Need Fix - add empty line at the end of file
```suggestion
	$(MAKE) -C $(BUILD_KERNEL) M=$(CURDIR) clean
	
```